charts/registry-backend/values.yaml

```diff
@@ -86,10 +86,9 @@ service:
 
 ingress:
   enabled: true
-  annotations:
-    kubernetes.io/ingress.class: traefik
-  routes:
-    - match: PathPrefix(`/api/`)
+  annotations: {}
+  host: sensrnet.local
+  path: /api
```

Comment:
No trailing slash?

Reply:
Deze ga ik goed in de gaten houden, snap eerlijk gezegd niet precies wanneer die nou wel en niet nodig is. Met het lokaal testen werkt het namelijk ook prima zonder. En als het niet nodig is gewoon maar weglaten lijkt me 😄